packages/components/organisms/f-cookie-banner/test-utils/component-objects/f-cookieBanner-legacy.component.js

```diff
@@ -1,10 +1,12 @@
 const Page = require('@justeat/f-wdio-utils/src/page.object');
 
-module.exports = class CookieBanner extends Page {
+class CookieBanner extends Page {
     get component () { return $('[data-test-id="legacyCookieBanner-component"]'); }
 
     get cookiePolicyLink () { return this.component.$('[data-test-id="cookie-policy-link"]'); }
 
+    get policyLinkHref () { return this.cookiePolicyLink.getAttribute('href'); }
```

Comment:
Might be worth matching the naming convention of this with the dependant element? e.g. `cookiePolicyLinkHref`?